src/components/UserProfile.re

```diff
@@ -227,79 +230,136 @@ module UserDetails = {
-          {optTwitter->reactMap(twitterHandle =>
-             <a
-               className=Styles.navListText
-               target="_blank"
-               rel="noopener noreferrer"
-               href={"https://twitter.com/" ++ twitterHandle}>
-               {("@" ++ twitterHandle)->restr}
-             </a>
-           )}
-          <br />
-          {optDescription->reactMap(description => <p> description->restr </p>)}
-          <a
-            className=Styles.navListText
-            target="_blank"
-            rel="noopener noreferrer"
-            href={"https://" ++ etherScanUrl ++ "/address/" ++ userAddress}>
-            {Helper.elipsify(userAddress, 10)->restr}
-          </a>
-          <br />
-          // NOTE: the number of loyalty tokens of a user currently will always show.
-          //       We are thinking of making this "private" only to the current logged in user. To enable this remove the `|| true` from the line below
-          {isAddressCurrentUser
-             ? <>
-                 <small>
-                   <p>
-                     {(
-                        "Claimed Loyalty Token Balance: "
-                        ++ {
-                          totalLoyaltyTokensOpt->Option.mapWithDefault(
-                            "Loading", claimedLoyaltyTokens => {
-                            claimedLoyaltyTokens->Web3Utils.fromWeiBNToEthPrecision(
-                              ~digits=6,
-                            )
-                          });
-                        }
-                        ++ " WLT"
-                      )
-                      ->restr}
-                   </p>
-                 </small>
-                 {switch (currentlyOwnedTokens) {
-                  | [||] => React.null
-                  | currentlyOwnedTokens =>
-                    currentlyOwnedTokens
-                    ->Array.map(id =>
-                        <ClaimLoyaltyTokenButtons
-                          id
-                          key=id
-                          refreshLoyaltyTokenBalance=updateFunction
-                        />
-                      )
-                    ->React.array
-                  }}
-                 <a href="/#ethturin-quadratic-voting"> "vote"->restr </a>
-               </>
-             : <small>
-                 <p>
-                   {(
-                      "Loyalty Token Balance Generated: "
-                      ++ {
-                        totalLoyaltyTokensAvailableAndClaimedOpt->Option.mapWithDefault(
-                          "Loading", ((totalLoyaltyTokens, _)) => {
-                          totalLoyaltyTokens->Web3Utils.fromWeiBNToEthPrecision(
-                            ~digits=5,
-                          )
-                        });
-                      }
-                      ++ " WLT"
-                    )
-                    ->restr}
-                 </p>
-               </small>}
+          {switch (nonUrlState) {
```

Comment:
Added the "user verification screen" and "UpdateDepositScreen" to the left pannel.